src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -86,21 +86,22 @@ internal PublicClientApplicationBuilder WithBroker(bool enableBroker)
             return this;
         }
 
-#if !NET_CORE_BUILDTIME // include for other platforms and for runtime
-#if iOS
-
+#if !ANDROID_BUILDTIME && !WINDOWS_APP_BUILDTIME && !NET_CORE_BUILDTIME && !DESKTOP_BUILDTIME && !MAC_BUILDTIME
         /// <summary>
         /// 
         /// </summary>
         /// <param name="keychainSecurityGroup"></param>
         /// <returns></returns>
         public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keychainSecurityGroup)
         {
+#if iOS || NETSTANDARD1_3
```

Comment:
Please use NETSTANDARD flag. I believe NETSTANDARD1_3 will also work because msbuild will set it, but we'd rather use a the simpler version which we set on our own in MSAL.common.props

Reply:
resolved